homeassistant/components/generic_thermostat/climate.py

```diff
@@ -1,11 +1,12 @@
-"""Adds support for generic thermostat units."""
```

Comment:
These generic_thermostat changes should not be here.

Reply:
I think you need to rebase your branch on dev.

Reply:
I needed them to merge the tree with latest changes. Now resolved.